### ISSUE TITLE
Finish Trackman API

### ISSUE THREAD

**Issue by echarlie** [wuvt/trackman#28]
_From @mutantmonkey on July 11, 2016 0:54_

Trackman's API is incomplete. While it is possible to achieve this functionality by making HTTP requests, these features are not part of the API and do not have responses that are useful for applications outside of a web browser. It may make sense to move the UI for these completely to the client side in order to avoid duplication of effort.

The following API calls should be added:
- [x] Create new DJSet ("login" although there isn't really any authentication)
- [ ] Start automation
  - This makes more sense as an API call anyway...
- [x] Report track
  - Since no one actually manages these, it might make sense for this feature to be removed.
- [x] End DJSet
  - Perhaps add an additional function for this that is called in both places.
  - The email playlist parameter is passed directly to this function currently, so the API will need to support this as well.

Register new DJ could also be a useful API call, but isn't really an important missing API feature.

_Copied from original issue: wuvt/wuvt-site#207_

**Comment by echarlie** [comment]
@mutantmonkey this can be marked as complete, yes?

**Comment by mutantmonkey** [comment]
No, the checklist is accurate. There is still no API call to start automation.
